desktop/src/handlers/moduleHandler.js

```diff
@@ -177,7 +177,14 @@ class ModuleHandler {
         bridge.send(startProgressBar(name, 0))
 
         try {
-          npm.run(['install', '-S', `${name}@${version}`], {cwd: installPath}, (err) => {
+          const command = [
+            'install', '-S', `${name}@${version}`,
+            ...options.registry && ['--registry', options.registry]
```

Comment:
What does `...options.registry` do here? Should it just be `options.registry` without the `...`?